### PR TITLE
Make Close buttons consistent

### LINKED ISSUE #916: Ensure dialog "exit" buttons are consistently styled
### Dev Home version

0.137.141.0

### Windows build number

10.0.22621.1778

### Other software

OS Build Version: 10.0.22621.1778.amd64fre.ni_release.220506-1250
.NET Version: .NET 6.0.16

### Steps to reproduce the bug

Nella dashboard, cliccando il tasto di aggiunta del widget

### Expected result

Ha fatto il suo lavoro, tuttavia sto segnalando qualche inconsistenza estetica.

### Actual result

La linea active che indica la sezione attuale non è al bordo della sezione selezionata, è all'interno. Inoltre, il bottone di uscita della 'modal' per l'aggiunta dei widget dovrebbe avere più margin rispetto alla topbar.

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by krschau**
## Summary of the pull request
We use a Close button ("X" button") in a few places in Dev Home. This change creates a CloseButton control so that styling (Margin, Padding, Alignment, Text) will be consistent throughout. Also remove FontIconSource and FontIconClose from SetupToolStyles.xaml, which are not used anywhere.

## References and relevant issues
#916

## PR checklist
- [x] Closes #916
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: common/Views/CloseButton.xaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Style="{StaticResource DefaultButtonStyle}"
+    HorizontalAlignment="Right"
+    VerticalAlignment="Top"
+    BorderThickness="0"
+    Background="Transparent"
+    Padding="10">
```
Comment (guimafelipe) [inline common/Views/CloseButton.xaml:13]: We will need to add a localized AutomationProperties.Name to this control later, as the narrator will read it as "Button" and nothing else.

**Review by guimafelipe — APPROVED** [review]

**Comment by dhoehna** [comment]
Can the common close button view be used in the setuptool project?  That has a close button.

**Comment by krschau** [comment]
> Can the common close button view be used in the setuptool project? That has a close button.

@dhoehna  Can you point me to where you're talking about? Searching I see there's two other places where glyph E711 are referenced, but since they're in their own control, I wasn't sure I should mess with them.
``` 
                            <ctControls:SettingsCard.ActionIcon>
                                <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE711;"/>
                            </ctControls:SettingsCard.ActionIcon>
```

**Comment by dhoehna** [comment]
https://github.com/microsoft/devhome/blob/main/tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml#L66

I use the symbolIcon.

**Comment by krschau** [comment]
@dhoehna I updated SummaryView to use it. I tried it out myself and it looked good, but you may want to double check it. (The Padding is a little bigger in my control, but only by 5 pixels)

**Review by dhoehna — APPROVED** [review]

**Pull request merged**